internal/buildinfo/env_dev.go

```diff
@@ -0,0 +1,21 @@
+//go:build !production
+
+package buildinfo
+
+import (
+	"time"
+
+	"github.com/superfly/flyctl/internal/version"
+)
+
+var environment = "development"
+
+func loadBuildTime() error {
+	cachedBuildTime = time.Now()
```

Comment:
This seems odd, dev builds report that they were built when they're run?

Reply:
Oh yes good eye! This is a temporary hack to avoid disrupting the build scripts while we test stuff out. A subsequent PR will use the build time which _will_ be passed in after build scripts are updated.

Reply:
Just for reference, using `time.Now()` is [the current behavior](https://github.com/superfly/flyctl/pull/2864/files#diff-4e852a88a07db777820550fedc019767dbb71b3fa558f4e7a59a6bfcdfe76d15L192-L216). This just moves it to another place for now. It's odd, but made sense in actual dev because it was only ran on a local machine. I'd like to create another "environment" for these in-dev preview builds. But I don't know what to call them. Let me know if you have suggestions :D

Reply:
Maybe skunkworks, in keeping with the fly/R&D themes?

Reply:
oh I like that